test/framework/src/main/java/org/elasticsearch/bootstrap/BootstrapForTesting.java

```diff
@@ -101,7 +101,7 @@ public class BootstrapForTesting {
         // check for jar hell
         try {
             final Logger logger = LogManager.getLogger(JarHell.class);
-            JarHell.checkJarHell(logger::debug);
+            // JarHell.checkJarHell(logger::debug);
```

Comment:
Do we want to uncomment this?

Reply:
Yes please, don't comment this out. :)

Reply:
Damn it.

Reply:
There's something wrong with the security test that it isn't worth me fixing because I never hack on it. So I just commented it out. And committed it twice. This is what NOCOMMIT is for, children.